projects/js-packages/charts/src/components/pie-semi-circle-chart/pie-semi-circle-chart.tsx

```diff
@@ -275,7 +287,7 @@ const PieSemiCircleChartInternal: FC< PieSemiCircleChartProps > = ( {
 			) }
 
 			{ showLegend && (
-				<Legend
+				<BaseLegend
```

Comment:
I wonder why chartId is not passed here, which is not consistent with the implementation in line chart